### DISCUSSION TITLE
Add translation in xDrip+

### DISCUSSION THREAD

**Discussion by lz2jvc** [NightscoutFoundation/xDrip#3715] [General]
I wrote about this topic nearly a year ago about the locked xDrip+ menu texts, which I didn't have access to translate until then. Consequence things happened, but still left a small part of the menu locked without access to translation for the different languages. I wrote to Phil Thompson who said that these texts are waiting in the queue for release. He also said that he can't do anything else and that I wrote to Jamorham in crowdin, who he says only deals with the text for xDrip+ translations. That's what I did, but there was no response from him. By the way, with the last update of xDrip+ from 11.10.2024. there was the addition of new texts in the menu, but very interesting that they were immediately released for translation, but what I have been struggling to translate for almost 1 year is still in the queue for unlocking. I found out from Navid Fo that Navid200 can also help with this. I am providing pictures for the xDrip+ menu texts in question which are still untranslatable. Please help to make things OK.

![2024-01-03 16 26 16](https://github.com/user-attachments/assets/e7a40540-b334-45a7-9659-d8942b408cab)
![2024-01-03 16 29 15](https://github.com/user-attachments/assets/3b6ad459-d74a-45c7-a730-a7c82c5424c5)
![2024-01-03 16 31 14](https://github.com/user-attachments/assets/e05dbc7f-9323-40b0-9386-a4b2b0b03d81)
![2024-01-03 16 35 17](https://github.com/user-attachments/assets/37692cc8-a768-40de-8b33-830f4d564c98)
![2024-01-03 16 37 15](https://github.com/user-attachments/assets/7b971d75-bc51-42be-bd01-901591a55a18)
![2024-01-03 16 38 21](https://github.com/user-attachments/assets/59fb9087-a210-430a-b907-6db0e62ecb9a)
![2024-10-01 13 30 35](https://github.com/user-attachments/assets/7618d8e9-31b7-4291-9502-b072c9b22f2b)
![2024-01-03 16 17 32](https://github.com/user-attachments/assets/2a3ae3f1-2411-4ee3-a84c-3962efe8d031)
![335428937_233289115833418_4845173596349309533_n](https://github.com/user-attachments/assets/5e1978b6-338f-4faf-b475-14cb08ad1cdd)
![не](https://github.com/user-attachments/assets/3a7fe817-09bf-43f5-a122-a33cc7d89fec)

**Comment by Navid200** [comment]
Thanks for reporting this

Translation is a subject that sometimes can get very touchy.  Not everyone agrees what should and what shouldn't be translated.  
As an example, I don't believe it is a very good idea to translate everything.
I worry about what could go wrong.  

I'm not sure it is such a good idea to translate insulin as an example.

---  
Sometimes, someone opens a submission with several changes.  If only one of them raises a question, the entire submission will be placed on hold.  So, it is always best to make changes in very small steps.  
  
---  
I will use the images you have posted (thanks a lot) and open a submission to translate a few.  I hope they will be accepted.
We can take it from there depending on what happens.  
  
---  
Thanks for having been patient and bringing up something that you did a year ago again.

**Reply by lz2jvc** [reply]
It's been several xDrip+ updates and almost a month since my question was posted here. So far I don't see any development on this yet. And I have another question I posted a long time ago in the xDrip+ group about a bug in entering a value when your keyboard is in Cyrillic.  And here in Github the same question has long been posed by another person, namely https://github.com/nightscoutfoundation/xdrip/pull/2998?fbclid=iwy2xjawfoygrua2FLBQIXMAITE8 QN5Q1Mimquiw69vy5iv_aem_tpzt9saqmtyukz33tdjcqw . Both issues are still pending. How long does it take to receive the items?

**Reply by Navid200** [reply]
This PR was opened to address your request:  https://github.com/NightscoutFoundation/xDrip/pull/3716  

We have to be patient for the review process.

I respectfully ask you to be patient and understanding of the work that goes into maintaining this repository.  
The lead developer has to find time to review submissions.  He is not paid for this.  He has been doing this for years.  He already has a job and family and has all the pressure every one of us has in our lives.  There is no reason to make him feel criticism after everything he has done.  

I hope you will consider yourself being a part of this team.
You have been providing feedback and I value it.  I have already worked with you to address issues that you have detected and have raised.  I am at fault for not having acknowledged your contributions.  I am sorry about that.  We appreciate and need your support and contributions and hope that you continue.
I promise that your posts are taken seriously and I always pursue them to get them addressed as much as possible.

Please feel free to continue to raise issues and share your feelings.  But, please don't use a critical tone.  
Thanks for your understanding and patience.

**Reply by Navid200** [reply]
If you attempt to enter a value for the persistent high alert threshold, do you experience the same problem?
